aws_google_auth/prepare.py

```diff
@@ -18,9 +18,10 @@ def default_if_none(value, default):
         return value if value is not None else default
 
     google_config.profile = default_if_none(profile, google_config.profile)
-
     _create_base_aws_cli_config_files_if_needed(google_config)
-    _load_google_config_from_stored_profile(google_config, google_config.profile)
+
+    if google_config.profile is not None:
```

Comment:
Why the not none? Wouldn't this ignore second calls when no profile is supplied at cli?

I would expect if you didn't pass one first time, it would store a none and therefore stay none.

Or am I overthinking it.  Need to test the code when actually on a computer.